app/helpers/mailer_helper.rb

```diff
@@ -110,6 +110,34 @@ def creation_title(creation)
     end
   end
 
+  # e.g., 1 word or 50 words
```

Comment:
We do something quite similar to all of this in the share code:

```
<a href="https://archiveofourown.org/works/###"><strong>Title</strong></a> (x words) by <a href="https://archiveofourown.org/users/user"><strong>creator</strong></a>
```

So I thought about not putting this in the mailer helper and not using mailer keys for the translations... but I decided I didn't have much faith in the assumption the mailers and share code would always match (whereas having similar text with similar formatting throughout the mailers is something we should always strive for). The share code should probably just be handled as its own separate thing.

Reply:
Let's make `creation_word_count` private, so no one tries to use it elsewhere.